xbmc/guilib/guiinfo/GUIInfoLabels.h

```diff
@@ -564,6 +564,11 @@
 #define SYSTEM_BUILD_VERSION_CODE 1007
 #define SYSTEM_BUILD_VERSION_GIT 1008
 
+static constexpr int SYSTEM_LOCALE_TIMEZONECOUNTRY = 1009;
```

Comment:
Unsigned int would probably be more appropriate? These don't go negative (nor even to 0)

Reply:
@ksooo specifically requested a type int, but I'm happy to change to unsigned int if required.

Reply:
Required? No, but I appreciate that our team brings different opinions to the table and isn't afraid to challenge existing assumptions. @ksooo what do you think?

Reply:
@garbear you are right, `unsigned int` is a better fit.

Reply:
Done.